tests/lib/rules/object-curly-spacing.js

```diff
@@ -811,6 +811,17 @@ ruleTester.run("object-curly-spacing", rule, {
                 }
             ]
         },
+        {
+            code: "var foo = { /* comment */ };",
+            output: "var foo = {};",
```

Comment:
Oh no, that's no good. We don't want to delete the comment! I'll work on either removing the spaces and leaving the comment or skipping the fix in this case.